pod/perlop.pod

```diff
@@ -2862,6 +2859,52 @@ on the terminating line.
 If the terminating string is quoted, the type of quotes used determine
 the treatment of the text.
 
+ my $person = 'John';
+
+ print uc << "EOT";
+ Hello, $person!  
+ And the text goes on.
+ EOT
+
+This yields
+
+ HELLO, JOHN!  
+ AND THE TEXT GOES ON.
+
+The same result happens if the double quotes are omitted 
+
+ my $person = 'John';
+ print uc <<EOT;
+ Hello, $person!  
+ And the text goes on.
+ EOT
+
+ HELLO, JOHN!  
+ AND THE TEXT GOES ON.
+
+If instead, single quotes are used, we get
+
+ print uc <<'EOT';
+ Hello, $person!  
+ And the text goes on.
+ EOT
+
+ my $person = 'John';
```

Comment:
I think this line meant to precede the here-doc declaration (to be a consistent example with the others).